description: search for {initial,master}_preferences in /etc/chromium
author: Andres Salomon <dilinger@debian.org>

The default chromium behavior of checking the current binary directory for
initial_preferences or master_preferences doesn't conform to debian policy.

--- a/chrome/browser/first_run/first_run_internal_linux.cc
+++ b/chrome/browser/first_run/first_run_internal_linux.cc
@@ -19,10 +19,7 @@ bool IsOrganicFirstRun() {
 }
 
 base::FilePath InitialPrefsPath() {
-  // The standard location of the initial prefs is next to the chrome binary.
-  base::FilePath initial_prefs;
-  if (!base::PathService::Get(base::DIR_EXE, &initial_prefs))
-    return base::FilePath();
+  base::FilePath initial_prefs = base::FilePath("/etc/chromium");
 
   base::FilePath new_path = initial_prefs.AppendASCII(installer::kInitialPrefs);
   if (base::PathIsReadable(new_path))
